Crash when config file does not exist

When we run the binary specifying a config file that doesn't exist, we get a crash.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x49b555]

goroutine 1 [running]:
bytes.(*Buffer).WriteTo(0xc0000972f0, 0x0, 0x0, 0x0, 0x0, 0xc0000c4940)
	/home/travis/.gimme/versions/go1.14.6.linux.amd64/src/bytes/buffer.go:239 +0x85
github.com/rs/zerolog.ConsoleWriter.Write(0x0, 0x0, 0x0, 0x0, 0x0, 0xc0000c4940, 0x4, 0x4, 0x0, 0x0, ...)
	/home/travis/gopath/src/github.com/rs/zerolog/console.go:116 +0x349
github.com/rs/zerolog.levelWriterAdapter.WriteLevel(...)
	/home/travis/gopath/src/github.com/rs/zerolog/writer.go:20
github.com/rs/zerolog.(*Event).write(0xc0000b2720, 0x3f, 0x1f4)
	/home/travis/gopath/src/github.com/rs/zerolog/event.go:76 +0x122
github.com/rs/zerolog.(*Event).msg(0xc0000b2720, 0xc00009a2d0, 0x4b)
	/home/travis/gopath/src/github.com/rs/zerolog/event.go:140 +0x210
github.com/rs/zerolog.(*Event).Msgf(0xc0000b2720, 0x756944, 0x1e, 0xc00010fde8, 0x1, 0x1)
	/home/travis/gopath/src/github.com/rs/zerolog/event.go:127 +0x83
github.com/issmirnov/docker-tag-yoinker/config.LoadConfig(0x7ffd377a0994, 0xf, 0x9d1ac0, 0x2, 0x0)
	/home/travis/gopath/src/github.com/issmirnov/docker-tag-yoinker/config/config.go:15 +0x20c
main.main()
	/home/travis/gopath/src/github.com/issmirnov/docker-tag-yoinker/main.go:52 +0x1d4
```

I need to check the LoadConfig method and ensure the file exists.

--- a/.idea/docker-tag-yoinker.iml
+++ b/.idea/docker-tag-yoinker.iml
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="WEB_MODULE" version="4">
+  <component name="Go" enabled="true" />
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
     <orderEntry type="inheritedJdk" />

--- a/config/config.go
+++ b/config/config.go
@@ -6,7 +6,6 @@ import (
 	"github.com/davecgh/go-spew/spew"
 	"github.com/issmirnov/docker-tag-yoinker/interfaces"
 	"github.com/pelletier/go-toml"
-	"github.com/rs/zerolog/log"
 )
 
 // LoadConfig takes a file name and the global context. It will save the parsed
@@ -14,7 +13,7 @@ import (
 func LoadConfig(fname string, ctx *interfaces.Context) (err error) {
 	doc, err := ioutil.ReadFile(fname)
 	if err != nil {
-		log.Error().Msgf("Could not read config file: %s", err.Error())
+		ctx.Logger.Error().Msgf("Could not read config file: %s", err.Error())
 		return
 	}
 

--- a/go.mod
+++ b/go.mod
@@ -1,11 +1,18 @@
 module github.com/issmirnov/docker-tag-yoinker
 
-go 1.14
+go 1.17
 
 require (
+	github.com/BurntSushi/toml v0.3.1
 	github.com/Masterminds/semver v1.5.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/pelletier/go-toml v1.8.0
 	github.com/rs/zerolog v1.19.0
 	github.com/smartystreets/goconvey v1.6.4
+)
+
+require (
+	github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1 // indirect
+	github.com/jtolds/gls v4.20.0+incompatible // indirect
+	github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d // indirect
 )

